hack/jenkins/e2e.sh

```diff
@@ -192,6 +192,19 @@ case ${JOB_NAME} in
     MINION_SIZE="n1-standard-1"
     NUM_MINIONS="2"
     ;;
+
+  # Runs non-flaky tests on GCE on the release-0.21 branch,
+  # sequentially. As a reminder, if you need to change the skip list
+  # or flaky test list on release-0.21, you'll need to propose a pull
+  # request directly to release-0.21.
+  kubernetes-e2e-gce-release-0.21)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e"}
```

Comment:
Yeah, let's pretend I didn't just cut and paste this ... badly. Sigh. Re-doing this again.